### PR TITLE
[PT-143]-JsonFormat

### PR DESCRIPTION
**Pull request by ItsFaizan**
# [PT-143](https://0xbase.atlassian.net/browse/PT-143?atlOrigin=eyJpIjoiODQ1YTA3NWNjMDNhNDVjZGJmMjJmMGIwNDhhNjgxMTciLCJwIjoiaiJ9) - Json Format

### Address and bundles saved in Json format

[PT-143]: https://0xbase.atlassian.net/browse/PT-143?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/containers/walletForm/WalletForm.tsx [hunk 1]
```diff
@@ -34,6 +34,11 @@ const WalletForm = () => {
       localStorage.setItem('bundles', JSON.stringify(bundles))
     }
   }, [addresses, bundles])
+
+  useEffect(() => {
+    console.log('Addresses:', addresses)
+    console.log('Bundles:', bundles)
+  }, [])
```
Comment (fuddin95) [inline src/containers/walletForm/WalletForm.tsx:41]: what is the purpose of this useEffect?

File: src/containers/walletForm/WalletForm.tsx [hunk 2]
```diff
@@ -73,6 +78,26 @@ const WalletForm = () => {
...
+
+      if (groupedAddresses[type]) {
+        groupedAddresses[type].push(addressValue)
+      } else {
+        groupedAddresses[type] = [addressValue]
+      }
+    })
+
+    return groupedAddresses
+  }
```
Comment (fuddin95) [inline src/containers/walletForm/WalletForm.tsx:96]: better place for this function is in Utils

File: src/containers/walletForm/WalletForm.tsx (hunk 1 above)
Comment (ItsFaizan) [reply src/containers/walletForm/WalletForm.tsx:41]: It was just for testing, to check previous addresses and bundles values. Removed it

File: src/containers/walletForm/WalletForm.tsx (hunk 2 above)
Comment (ItsFaizan) [reply src/containers/walletForm/WalletForm.tsx:96]: Done

**Review by fuddin95 — APPROVED** [review]

**Pull request merged**